packages/react-i18n/src/currencyCode.ts

```diff
@@ -150,12 +134,30 @@ export enum CurrencyCode {
   Uyu = 'UYU',
   Uzs = 'UZS',
   Vuv = 'VUV',
-  Ves = 'VES',
   Vnd = 'VND',
   Xof = 'XOF',
   Yer = 'YER',
   Zmw = 'ZMW',
-  // Codes after this line are deprecated
```

Comment:
> I'm not quite sure about having this `CurrencyCode` type at all in this package, TBH. If this wasn't a public repo, I'd work at removing this type from this package altogether, because it isn't used within the package, and it only creates more points of maintenance.

I agree that fewer copies of the same data is a good goal.

> I'm curious @cejaekl if you have any strong opinions about preserving the deprecated comment, since you [introduced it](https://github.com/Shopify/quilt/pull/1632), though that was a while ago.

I don't have a strong opinion on this.  The risk is that someone at some point will present the user with the complete list of currencies given here, which won't make sense because some currencies are no longer in use.  The comment was intended as a warning that folks should consider that, no more.